packages/honeycomb-opentelemetry-web/src/global-errors-autoinstrumentation.ts

```diff
@@ -30,12 +30,16 @@ export function getStructuredStackTrace(error: Error | undefined) {
   const functions: string[] = [];
   const urls: string[] = [];
 
-  for (const stackFrame of structuredStack) {
+  if (!structuredStack) {
```

Comment:
Nit: If we want to be extra safe here we might also want to check that `structuredStack` is iterable.

Reply:
💯 
[28a6ac4](https://github.com/honeycombio/honeycomb-opentelemetry-web/pull/442/commits/28a6ac4ab2b50e76b848af52b0c34f13497c0f84)